src/sections/pools/farms/modals/joinedFarmDetails/JoinedFarmsDetails.tsx

```diff
@@ -0,0 +1,83 @@
+import { Button } from "components/Button/Button"
+import { Modal } from "components/Modal/Modal"
+import { Text } from "components/Typography/Text/Text"
+import { useState } from "react"
+import { useTranslation } from "react-i18next"
+import { FarmDetailsModal } from "../details/FarmDetailsModal"
+import { FarmDetailsCard } from "../../components/detailsCard/FarmDetailsCard"
+import { ClaimeRewardsCard } from "../../components/claimableCard/ClaimeRewardsCard"
+
+type JoinedFarmsDetailsProps = {
+  isOpen: boolean
+  onClose: () => void
+  availableFarms: any
```

Comment:
Can we avoid usage of `any` here? I find it hard to verify if the data is correctly passed around, esp. if I am not sure, where `any` is used

Reply:
It is only fot test data. I'll add types after we connect data from a chain

Reply:
How can we make sure that after we connect data from chain, there won't be any `any` left?

Reply:
We can make sure when we review epic-farms branch as a whole

Reply:
Also, when we start connecting data from a chain, you will see what data ts suggest you. S, I think it shouldn't be a problem

Reply:
Ok, let's also enable / add an eslint rule so that all of the any is caught